packages/core/src/deserializer.ts

```diff
@@ -31,6 +33,7 @@ const byIdMap = <T extends { id: Id }>(ts: T[]): { [id: Id]: T } => {
 export const deserializeSerializationChunk = <NT extends Node>(
     serializationChunk: SerializationChunk,
     instantiationFacade: InstantiationFacade<NT>,
+    primitiveTypeDeserializer: PrimitiveTypeDeserializer,
```

Comment:
Sorry for not realizing this earlier: this would break the API of the package. Is it possible to move this argument to the last position, and have it have a default value?

Reply:
Good idea, done

Reply:
Uhm, I'm seeing that argument still in the 3rd position (and without a default value).